src/pool/srv_target.c

```diff
@@ -516,7 +516,7 @@ aggregate_pool_space(struct daos_pool_space *agg_ps,
 	D_ASSERT(agg_ps && ps);
 
 	if (ps->ps_ntargets == 0) {
-		D_ERROR("Skip emtpy space info\n");
+		D_DEBUG(DB_TRACE, "Skip emtpy space info\n");
```

Comment:
Isn't this an unexpected error? Why it's changed from error to debug?

Reply:
because now for pool connect it is optional to query the space info, so in ds_pool_tgt_connect_handler it possible needs not the collective call pool_tgt_query, then in ds_pool_tgt_connect_aggregator possibly with zero content pool space info.